.travis.yml

```diff
@@ -101,6 +101,13 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
       presto-hive-hadoop2/bin/run_hive_tests.sh
+      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID && -v S3_TESTS_AWS_SECRET_ACCESS_KEY && -v S3_TESTS_BUCKET_ENDPOINT && -v S3_TESTS_BUCKET ]]; then
```

Comment:
Maybe we could have some conditional build like https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

That way it will be known that s3 tests were executed and they were passed. Now in case of misconfiguration it could be hard to notice that s3 tests were not executed.

Reply:
i am not entirely sure this is a good idea. Let's keep it simple, as is

Reply:
However, it should be enough to check one variable for existence (or use OR).